test/unit/carriers/ups_test.rb

```diff
@@ -332,6 +332,21 @@ def test_label_request_prepay
     refute_empty prepay
   end
 
+  def test_label_request_bill_third_party
+    response = Nokogiri::XML @carrier.send(:build_shipment_request,
+                                           location_fixtures[:beverly_hills],
+                                           location_fixtures[:annapolis],
+                                           package_fixtures.values_at(:chocolate_stuff),
+                                           :test => true,
+                                           :bill_third_party => true,
+                                           :billing_account=>"A01B24",
+                                           :billing_zip=>"97013",
+                                           :billing_country => "US")
+
+    bill_third_party = response.search '/ShipmentConfirmRequest/Shipment/PaymentInformation/BillThirdParty'
+    refute_empty bill_third_party
```

Comment:
I would also test that at least one of the options is being correctly passed through into the XML.